core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/impl/AttachmentCategory.java

```diff
@@ -0,0 +1,88 @@
+package uk.ac.bbsrc.tgac.miso.core.data.impl;
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import com.eaglegenomics.simlims.core.SecurityProfile;
+
+import uk.ac.bbsrc.tgac.miso.core.data.Aliasable;
+import uk.ac.bbsrc.tgac.miso.core.data.Deletable;
+
+@Entity
+public class AttachmentCategory implements Aliasable, Deletable, Serializable {
+
+  private static final long serialVersionUID = 1L;
+
+  private static final long UNSAVED_ID = 0;
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.AUTO)
+  private long id = UNSAVED_ID;
```

Comment:
This is `categoryId` in the SQL migration, which might be causing the test failures.